### DISCUSSION TITLE
Facing issue in Running Cypress test case in Jenkin and View Report in BrowserStack

### DISCUSSION THREAD

**Discussion by cypress1980** [cypress-io/cypress#15918] [CI setup] [answered]
I am  Running Cypress test case in Jenkin and Want to see the result in BrowserStack

Browserstack.json screens shot
![image](https://user-images.githubusercontent.com/76464201/114273465-c1c93580-9a37-11eb-967d-22152ae3fe90.png)

Jenkin Error screen shot
![image](https://user-images.githubusercontent.com/76464201/114273505-e7563f00-9a37-11eb-8f49-e3eea678d226.png)

Jenkin configuration screen shot

![image](https://user-images.githubusercontent.com/76464201/114273533-048b0d80-9a38-11eb-8865-2106e96b2724.png)

**Comment by bahmutov** [comment, marked as answer]
The error is not coming from the Cypress Test Runner, and it is not coming from Cypress Dashboard, but seems it is coming from the BrowserStack itself. Please follow up with them to resolve it.
